exporter/f5cloudexporter/config_test.go

```diff
@@ -66,6 +66,7 @@ func TestLoadConfig(t *testing.T) {
 				Headers: map[string]string{
 					"User-Agent": "opentelemetry-collector-contrib {{version}}",
 				},
+				Compression: "gzip",
```

Comment:
@gramidt please confirm this change is ok, it is a result of https://github.com/open-telemetry/opentelemetry-collector/commit/5470057bdfacd9821572bbb2c7cd00e5e4fae8e9